release-notes/opensearch-knn.release-notes-2.1.0.0.md

```diff
@@ -0,0 +1,25 @@
+## Version 2.1.0.0 Release Notes
+
+Compatible with OpenSearch 2.1.0
+
+### Enhancements
+
+* Include scripted metric context to allow list ([#421](https://github.com/opensearch-project/k-NN/pull/421))
+
+### Bug Fixes
+
+* Change VectorReaderListener to expect number array ([#416](https://github.com/opensearch-project/k-NN/pull/416))
+
+### Infrastructure
+ 
+* Update opensearch version in BWCWorkflow ([#402](https://github.com/opensearch-project/k-NN/pull/402))
+* Adding workflow for creating documentation issues ([#403](https://github.com/opensearch-project/k-NN/pull/403))
+* Add Querying Functionality to OSB ([#409](https://github.com/opensearch-project/k-NN/pull/409))
+* Add OpenSearch Benchmark index workload for k-NN ([#364](https://github.com/opensearch-project/k-NN/pull/364))
+* Set tests.security.manager flag to false in integTestRemote task
+([#410](https://github.com/opensearch-project/k-NN/pull/410))
```

Comment:
is this formatting correct? I do see it as split into 2 lines, but I'm not sure if that's the github viewer or not

Reply:
<img width="426" alt="Screen Shot 2022-06-30 at 11 06 48 AM" src="https://user-images.githubusercontent.com/9303427/176747353-c65ef54b-5c6a-4a53-b563-d6ed4b6617e6.png">

Looks fine in the editor : )

Reply:
It is just link, it wont render on ui , but we can make it into 1 line as well. 
https://github.com/Shivamdhar/k-NN/blob/release-notes-for-2.1/release-notes/opensearch-knn.release-notes-2.1.0.0.md